bears/jinja2/Jinja2Bear.py

```diff
@@ -0,0 +1,337 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.Diff import Diff
+
+
+def generate_spacing_diff(file, filename, line, line_number,
+                          match_object, required_spacing):
+    '''
+    Generate a diff for incorrectly spaced control or variable tags.
```

Comment:
I feel like this function would really like a doctest example! Makes it way better understandable and it's easily viewable :)